shared/Microsoft.Extensions.Certificates.Generation.Sources/CertificateManager.cs

```diff
@@ -0,0 +1,435 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace Microsoft.Extensions.Certificates.Generation
```

Comment:
Nit: seems like this should be `Microsoft.AspNetCore.Certificates.Generation` given that all the constants below say "aspnetcore ...." in them.

Reply:
It doesn't depend on any ASP.NET specific library, which I believe is the main driving factor for deciding between AspNetCore and Extensions.

Reply:
Nate's right, this is an ASP.NET component because it is all about ASP.NET Certificates as shown by the constants below.